plugin.video.putio/main.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+
+# put.io kodi addon
+# Copyright (C) 2009  Alper Kanat <alper@put.io>
+# Copyright (C) 2016  Put.io Developers <devs@put.io>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import urlparse
+import requests
+
+import xbmc
+import xbmcgui
+import xbmcplugin
+
+from resources.lib.helper import __lang__
+from resources.lib.helper import __settings__
+from resources.lib.helper import PutioApiHandler
+from resources.lib.helper import PutioAuthFailureException
+
+__url__ = sys.argv[0]  # base URL ('plugin://plugin.video.putio/')
```

Comment:
In a future update, please remove the use of \__name\__ type labels. These are reserved for python internals and should not be used.